docker/Tomcat/webapp/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN set -eux && \
 	mkdir -p /opt/frank/secrets && \
 	# Create tomcat user for stepping down from root
 	groupadd -g ${GID} tomcat && \
-	useradd -u ${UID} -g tomcat -s /usr/sbin/nologin -l tomcat && \
+	useradd -u ${UID} -g tomcat -s /usr/sbin/nologin -l tomcat -d /usr/local/tomcat && \
```

Comment:
What is the value of `CATALINA_HOME`, referenced below? Should that perhaps be used instead?